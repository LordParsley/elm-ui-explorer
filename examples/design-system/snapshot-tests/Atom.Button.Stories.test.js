
    const tree = `<div>
    <div class="snapshot" id="LargePrimary">
        <button class="button is-white" style="backgroundColor:#db7093;color:hsl(0, 0%, 96%);width:14em;">
            Primary
        </button>
    </div>
    <div class="snapshot" id="TinyPrimary">
        <button class="button is-white" style="backgroundColor:#db7093;color:hsl(0, 0%, 96%);width:10em;">
            Primary
        </button>
    </div>
    <div class="snapshot" id="LargeSecondary">
        <button class="button is-white" style="backgroundColor:#ff6347;color:hsl(0, 0%, 96%);width:14em;">
            Secondary
        </button>
    </div>
    <div class="snapshot" id="TinySecondary">
        <button class="button is-white" style="backgroundColor:#ff6347;color:hsl(0, 0%, 96%);width:10em;">
            Secondary
        </button>
    </div>
</div>`;
    test("Atom.Button.Stories", () => {
      expect(tree).toMatchSnapshot();
    });
    