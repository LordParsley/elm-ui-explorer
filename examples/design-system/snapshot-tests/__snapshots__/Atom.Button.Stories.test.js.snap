// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atom.Button.Stories 1`] = `
"<div>
    <div class=\\"snapshot\\" id=\\"LargePrimary\\">
        <button class=\\"button is-white\\" style=\\"backgroundColor:#db7093;color:hsl(0, 0%, 96%);width:14em;\\">
            Primary
        </button>
    </div>
    <div class=\\"snapshot\\" id=\\"TinyPrimary\\">
        <button class=\\"button is-white\\" style=\\"backgroundColor:#db7093;color:hsl(0, 0%, 96%);width:10em;\\">
            Primary
        </button>
    </div>
    <div class=\\"snapshot\\" id=\\"LargeSecondary\\">
        <button class=\\"button is-white\\" style=\\"backgroundColor:#ff6347;color:hsl(0, 0%, 96%);width:14em;\\">
            Secondary
        </button>
    </div>
    <div class=\\"snapshot\\" id=\\"TinySecondary\\">
        <button class=\\"button is-white\\" style=\\"backgroundColor:#ff6347;color:hsl(0, 0%, 96%);width:10em;\\">
            Secondary
        </button>
    </div>
</div>"
`;
